docs/misc/FAQ.md

```diff
@@ -28,403 +28,680 @@ Before diving in deeper, we recommend that you check the following resources as
+Community members have created [Alph Richlist](https://alph-richlist.vercel.app/) that lists the top holding addresses, alongside Alph.Pro, a comprehensive website/tool that offers data on all things DeFi on Alephium, including NFTs, the Richlist, Liquidity pools and more.
 
 In addition, you can find the wrapped ALPH/erc20 holders at [etherscan.io](https://etherscan.io/token/0x590F820444fA3638e022776752c5eEF34E2F89A6#balances)
 
-### What is the minimum Transaction Fee?
+
+### **What is the minimum Transaction Fee?[​](https://docs.alephium.org/frequently-asked-questions#what-is-the-minimum-transaction-fee)**
 
 Currently, the minimum transaction fee is set at `0.002` ALPH to prevent network DoS attacks. In the future, this minimum fee can be lowered, with the lowest possible minimum transaction fee being `0.00000000000001` ALPH on Alephium. The exact fee charged depends on the number of inputs (UTXOs) and signers involved in the transaction.
 
-### How many Transactions Per Second (TPS) are possible on Alephium?
 
-Alephium Mainnet can currently support over 400 TPS with 16 shards. It can scale up over 10k TPS by increasing the number of shards as necessary.  
+### **How many Transactions Per Second (TPS) are possible on Alephium?[​](https://docs.alephium.org/frequently-asked-questions#how-many-transactions-per-second-tps-are-possible-on-alephium)**
+
+Alephium Mainnet can currently support over 400 TPS with 16 shards. It can scale up over 10,000+ TPS by increasing the number of shards as necessary. \
 Read more about [the concept of TPS](https://medium.com/@alephium/transactions-per-second-tps-f13217a49e39).
 
-### What is Alephium's smallest denomination?
+
+### **What is Alephium's smallest denomination?[​](https://docs.alephium.org/frequently-asked-questions#what-is-alephiums-smallest-denomination)**
 
 Alephium allows up to 18 decimals and its smallest denomination is called Phi. Phi is equivalent to `0.000000000000000001` ALPH, or `10^-18` ALPH, while 1 ALPH equals `10^18` Phi.
 
-## dApps
 
-### Is there a DEX on Alephium?
+## **dApps[​](https://docs.alephium.org/frequently-asked-questions#dapps)**
+
+
+### **Is there a DEX on Alephium?[​](https://docs.alephium.org/frequently-asked-questions#is-there-a-dex-on-alephium)**
 
 Alephium has a [DEX prototype](https://alephium.github.io/alephium-dex/#/swap) running on Testnet. Learn more about it in [the DEX prototype article](https://medium.com/@alephium/dex-prototype-live-on-testnet-bac5e7d095ce).
 
 The [DEX contracts](https://github.com/alephium/alephium-dex/tree/master/contracts) have been thoroughly tested by the core developers, making them reliable and efficient. They can easily be forked and used for projects.
 
-### Is there any dApps on Alephium?
+Alephium’s DeFi is continuously expanding and you can check all deployed DEXes on [Coingecko ](https://www.coingecko.com/en/coins/alephium) and on [Alph.land](alph.land).
```

Comment:
There's an empty space in `[Coingecko ]`

Reply:
fixed. thanks